### PR TITLE
- shell variable substitution (docker configs)

### PR DESCRIPTION
**Pull request by AndrianBdn**
cc @aka-toxa @melya

### CODE REVIEW

**Comment by aka-toxa** [comment]
:+1:

**Comment by AndrianBdn** [comment]
:-1:  :cry:

**Comment by AndrianBdn** [comment]
hopefully fixed

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "configuration",
     "conf"
   ],
-  "author": "Petrovskyi Anatolii (Readdle)",
+  "author": "Readdle Inc.",
   "license": "MIT",
```
Comment (AndreyUtka) [inline package.json]: :+1:

**Pull request merged**